modules/swagger-codegen/src/main/resources/Java/libraries/okhttp-gson/ApiClient.mustache

```diff
@@ -2,10 +2,20 @@
 
 package {{invokerPackage}};
 
-import com.squareup.okhttp.*;
```

Comment:
The Google styleguide discourages wildcards:
https://google.github.io/styleguide/javaguide.html#s3.3.1-wildcard-imports 

If it's preferable to use wildcards from a codegen perspective, I can revert.